errai-navigation/src/main/java/org/jboss/errai/ui/nav/client/local/ContentDelegation.java

```diff
@@ -0,0 +1,36 @@
+package org.jboss.errai.ui.nav.client.local;
```

Comment:
Needs a license header. You can copy one from any other Java source file in the project. Just remember to update the year.